<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotForm</class>
 <widget class="QWidget" name="PlotForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>841</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="functionsGBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>321</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>321</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Functions</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="f1Label">
        <property name="text">
         <string>f1(x)=</string>
        </property>
        <property name="buddy">
         <cstring>f1LineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="f1LineEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QtColorComboBox" name="f1CCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="f2Label">
        <property name="text">
         <string>f2(x)=</string>
        </property>
        <property name="buddy">
         <cstring>f2LineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="f2LineEdit"/>
      </item>
      <item row="1" column="2">
       <widget class="QtColorComboBox" name="f2CCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="f3Label">
        <property name="text">
         <string>f3(x)=</string>
        </property>
        <property name="buddy">
         <cstring>f3LineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="f3LineEdit"/>
      </item>
      <item row="2" column="2">
       <widget class="QtColorComboBox" name="f3CCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="f4Label">
        <property name="text">
         <string>f4(x)=</string>
        </property>
        <property name="buddy">
         <cstring>f4LineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="f4LineEdit"/>
      </item>
      <item row="3" column="2">
       <widget class="QtColorComboBox" name="f4CCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="3">
    <widget class="FunctionPlotter" name="plotter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>281</width>
       <height>311</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="miscGBox">
     <property name="title">
      <string>Misc</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="exportPushButton">
        <property name="text">
         <string>&amp;Export to...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="controlGBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>321</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Control</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" rowspan="3">
       <widget class="QGroupBox" name="csGroupBox">
        <property name="title">
         <string>Coordinate Window</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="xminLabel">
           <property name="text">
            <string>x-min</string>
           </property>
           <property name="buddy">
            <cstring>xminDSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="xminDSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="minimum">
            <double>-99999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>99999.000000000000000</double>
           </property>
           <property name="value">
            <double>-10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="xmaxLabel">
           <property name="text">
            <string>x-max</string>
           </property>
           <property name="buddy">
            <cstring>xmaxDSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="xmaxDSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="minimum">
            <double>-99999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>99999.000000000000000</double>
           </property>
           <property name="value">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="yminLabel">
           <property name="text">
            <string>y-min</string>
           </property>
           <property name="buddy">
            <cstring>yminDSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="yminDSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="minimum">
            <double>-99999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>99999.000000000000000</double>
           </property>
           <property name="value">
            <double>-10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="ymaxLabel">
           <property name="text">
            <string>y-max</string>
           </property>
           <property name="buddy">
            <cstring>ymaxDSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="ymaxDSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="minimum">
            <double>-99999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>99999.000000000000000</double>
           </property>
           <property name="value">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QPushButton" name="standardPushButton">
           <property name="text">
            <string>Zomm To &amp;Standard</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1" rowspan="3">
       <widget class="QGroupBox" name="presentationGroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Presentation</string>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="xticksLabel">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>15</height>
            </size>
           </property>
           <property name="text">
            <string>x-Ticks</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="xticksDSpinBox">
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.200000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="xAutoCheckBox">
           <property name="text">
            <string>auto</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="yticksLabel">
           <property name="text">
            <string>y-Ticks</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="yticksDSpinBox">
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.200000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QCheckBox" name="yAutoCheckBox">
           <property name="text">
            <string>auto</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Grid</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QCheckBox" name="gridXCheckBox">
              <property name="text">
               <string>grid X</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="gridYCheckBox">
              <property name="text">
               <string>grid Y</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="gridBothCheckBox">
              <property name="text">
               <string>grid both</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtColorComboBox</class>
   <extends>QComboBox</extends>
   <header>qtcolorcombobox.hpp</header>
  </customwidget>
  <customwidget>
   <class>FunctionPlotter</class>
   <extends>QFrame</extends>
   <header>FunctionPlotter.hpp</header>
   <container>1</container>
   <slots>
    <signal>newXMin(double)</signal>
    <signal>newXMax(double)</signal>
    <signal>newYMin(double)</signal>
    <signal>newYMax(double)</signal>
    <signal>newXTicks(double)</signal>
    <signal>newYTicks(double)</signal>
    <slot>setXMin(double)</slot>
    <slot>setXMax(double)</slot>
    <slot>setYMin(double)</slot>
    <slot>setYMax(double)</slot>
    <slot>setXTicks(double)</slot>
    <slot>setYTicks(double)</slot>
    <slot>autoXTicks(bool)</slot>
    <slot>autoYTicks(bool)</slot>
    <slot>setStandardWindow()</slot>
    <slot>f1ToFile()</slot>
    <slot>toggleGrid(bool)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
